### PR TITLE
Packer customize package info

### PR DESCRIPTION
**Pull request by cyrilleverrier**

### CODE REVIEW

**Comment by cyrilleverrier** [comment]
@ruflin : Here is the first PR that allows customization of everything in the previous PR, except for the versionning

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -1,11 +1,14 @@
-
 ### VARIABLE SETUP ###
-
+#¤¤ Application using libbeat may override the following variables in their Makefile
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: Are chart 2 and 3 on purpose?

**Comment by tsg** [comment]
LGTM.

File: libbeat/scripts/Makefile (hunk 1 above)
Comment (cyrilleverrier) [inline libbeat/scripts/Makefile]: It was a typo error. Fixed in 5a13af4

**Comment by ruflin** [comment]
jenkins, test it

**Comment by ruflin** [comment]
jenkins, package it

**Comment by ruflin** [comment]
@cyrilleverrier Merged. Thanks.

**Comment by cyrilleverrier** [comment]
@ruflin thanks for merging. :+1:

**Pull request merged**